src/ripple/server/Port.h

```diff
@@ -107,6 +108,13 @@ operator<< (std::ostream& os, Port const& p)
             os << ip.to_string () << ", ";
     }
 
+    if (! p.secure_gateway_ip.empty ())
```

Comment:
This routine is getting big for an inline, consider moving the definition to `ripple/server/impl/Port.cpp`